packages/block-editor/src/components/iframe/content.scss

```diff
@@ -13,9 +13,9 @@
 	$inner-height: var(--wp-block-editor-iframe-zoom-out-inner-height);
 	$content-height: var(--wp-block-editor-iframe-zoom-out-content-height);
 	$prev-container-width: var(--wp-block-editor-iframe-zoom-out-prev-container-width);
-
-	transform: scale(#{$scale});
-
+	$container-width: var(--wp-block-editor-iframe-zoom-out-container-width, 100vw);
+	// Apply an X translation to center the scaled content within the available space.
+	transform: scale(#{$scale}) translateX(calc(( #{$prev-container-width} - #{ $container-width }) / 2 / #{$scale}));
```

Comment:
Move the scaled canvas to the right to occupy the space of the container.

Reply:
I think the `translateX()` could precede the `scale()` and the translation wouldn’t have to be manually counter-scaled but that’s a nit.

Reply:
We could also use individual [`scale`](https://developer.mozilla.org/en-US/docs/Web/CSS/scale) and [`translate`](https://developer.mozilla.org/en-US/docs/Web/CSS/translate) properties — if that helps changing them independently without rewriting the whole `transform`

Reply:
> I think the translateX() could precede the scale()

We tried that as well, but it seems like it's still necessary to apply the scaling to the `translateX`. 

> We could also use individual scale and translate properties

I'm going to try this, as I think we'll need to adjust the animating timings for this separately.